src/blocks/hueshift.rs

```diff
@@ -178,6 +259,8 @@ impl Default for HueshiftConfig {
                 Some(HueShifter::Gammastep)
             } else if has_command("hueshift", "wlsunset").unwrap_or(false) {
                 Some(HueShifter::Wlsunset)
+            } else if has_command("hueshift", "wl-gammarelay").unwrap_or(false) {
```

Comment:
Please check for either `wl-gammarelay` or `wl-gammarelay-rs`, since they have compatible DBus interfase.